examples/codesandbox-starter/src/index.js

```diff
@@ -0,0 +1,20 @@
+/** @jsx jsx */
+
+import ReactDOM from 'react-dom'
+import ThemeProvider, { Reset } from './theme'
+import { jsx, Layout, Styled as S } from 'theme-ui'
```

Comment:
Mind keeping the default names here? I think it's easier to read when the names used in JSX don't differ from the docs

```suggestion
import { jsx, Layout, Styled } from 'theme-ui'
```